com.github.Resticity.yml

```diff
@@ -0,0 +1,48 @@
+id: com.github.Resticity
+runtime: org.gnome.Platform
+runtime-version: '45'
+sdk: org.gnome.Sdk
+
+
+command: resticity
+finish-args:
+  - --share=ipc # Share IPC namespace with the host (necessary for X11).
+  - --socket=x11
+  - --device=dri # OpenGL rendering support.
+  - --talk-name=org.freedesktop.Notifications
+  - --filesystem=home
+  - --filesystem=host
+  - --socket=wayland
+  - --share=network
+  - --talk-name=org.kde.StatusNotifierWatcher
+
+build-options:
+  build-args:
+    - --share=network
+  env:
+    - GOROOT=/usr/lib/sdk/golang
+    - GOBIN=/app/go/bin
+    - CGO_ENABLED=0
+
+
+modules:
+  - shared-modules/libappindicator/libappindicator-gtk3-12.10.json
+  - ./webkit2.yaml
+  - ./restic.yaml
+  - ./rclone.yaml
+  - name: resticity
+    buildsystem: simple
+    build-commands:
+      - $GOROOT/bin/go install github.com/wailsapp/wails/v2/cmd/wails@v2.8.0
+      - PATH=$PATH:$GOROOT/bin $GOBIN/wails build -s
+      - install -Dm00755 build/bin/resticity $FLATPAK_DEST/bin/resticity
+      - install -Dm00644 packaging/resticity.png $FLATPAK_DEST/share/icons/hicolor/256x256/apps/resticity.png
+      - install -Dm00644 packaging/resticity.desktop $FLATPAK_DEST/share/applications/$FLATPAK_ID.desktop
+      - install -Dm00644 packaging/flatpak/$FLATPAK_ID.appdata.xml $FLATPAK_DEST/share/appdata/$FLATPAK_ID.appdata.xml
+    sources:
+      - type: dir
+        path: ../../
```

Comment:
end where does this `../..` lead to? It might work on your machine... definitely not on buildbot or another other.